src/components/exchange/ExchangeField.js

```diff
@@ -126,7 +128,7 @@ const ExchangeField = (
             ref={ref}
             testID={testID}
             useCustomAndroidMask={useCustomAndroidMask}
-            value={amount}
+            value={isInputFocused ? value : amount}
```

Comment:
can we copy the isFocused behavior in `ExhangeNativeField`?

Reply:
i feel like all that state logic is kinda unnecessary here

Reply:
hm could be but i'm not sure about the performance issues of calling `ref?.current?.isFocused()` all the time